File: .sdk/tm/php/utility/MakeContext.php
<?php
declare(strict_types=1);

// DungeonsAndDragonsTwo SDK utility: make_context

require_once __DIR__ . '/../core/Context.php';

class DungeonsAndDragonsTwoMakeContext
{
    public static function call(array $ctxmap, ?DungeonsAndDragonsTwoContext $basectx): DungeonsAndDragonsTwoContext
    {
        return new DungeonsAndDragonsTwoContext($ctxmap, $basectx);
    }
}

File: .sdk/tm/rb/utility/make_context.rb
# DungeonsAndDragonsTwo SDK utility: make_context
require_relative '../core/context'
module DungeonsAndDragonsTwoUtilities
  MakeContext = ->(ctxmap, basectx) {
    DungeonsAndDragonsTwoContext.new(ctxmap, basectx)
  }
end
